### ISSUE TITLE
Discard changes, unpublish, duplicate, and delete buttons not working in Studio 2.3.0

### ISSUE THREAD

**Issue by geoffreyjball** [sanity-io/sanity#2275]
The Discard Changes, Unpublish, Duplicate, and Delete buttons are not working in the Studio.

The Discard Changes, Unpublish, Duplicate, and Delete buttons are not responsive (i.e., they do not produce their normal behaviour).

1. Go the the editor page for any document.
2. Click the down arrow icon next to Publish.
3. Click any of the action buttons.

**Expected behavior**

The buttons would normally discard changes, unpublish the document, duplicate the document, and delete the document, respectively.

**Screenshots**

<img width="211" alt="Screen Shot 2021-02-01 at 1 39 18 PM" src="https://user-images.githubusercontent.com/45116528/106515691-45443480-6493-11eb-9caa-dea9ffe207be.png">

**Which versions of Sanity are you using?**

@sanity/cli             2.2.6 (up to date)
@sanity/base            2.3.0 (up to date)
@sanity/code-input      2.2.6 (up to date)
@sanity/components      2.2.6 (up to date)
@sanity/core            2.2.6 (up to date)
@sanity/default-layout  2.3.0 (up to date)
@sanity/default-login   2.2.6 (up to date)
@sanity/desk-tool       2.3.0 (up to date)
@sanity/vision          2.2.6 (up to date)

**What operating system are you using?**

macOS 11.1

**Which versions of Node.js / npm are you running?**

npm 7.4.0
node 15.2.0

**Additional context**

**Security issue?**

No

**Comment by rexxars** [comment]
Fixed in v2.3.1, thanks for reporting!

**Comment by AHBruns** [comment]
I am seeing this same issue using the following versions:
```
"dependencies": {
    "@sanity/base": "^2.10.2",
    "@sanity/components": "^2.2.6",
    "@sanity/core": "^2.10.2",
    "@sanity/default-layout": "^2.10.2",
    "@sanity/default-login": "^2.8.0",
    "@sanity/desk-tool": "^2.10.2",
    "@sanity/vision": "^2.10.0",
    "prop-types": "^15.7",
    "react": "^17.0",
    "react-dom": "^17.0",
    "sanity-plugin-datatable": "^1.1.0"
  },
```
Seems there's been a regression.
